Makefile

```diff
@@ -4,6 +4,17 @@ all: only_build only_test
 
 ensure:
 	cd misc/test && dep ensure -v
+	
+	# Install aws-iam-authenticator
+	# See: https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
+	curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.12.7/2019-03-27/bin/linux/amd64/aws-iam-authenticator
```

Comment:
It's important to point out here that these will install linux versions only rather than on macOS - maybe we can have a case based on us being on darwin?

Reply:
Hmm, this is a good point. This change was primarily intended for Travis CI testing, similarly to how we setup the `pulumi-eks` travis: https://github.com/pulumi/pulumi-eks/blob/master/.travis.yml#L12-L25.

Given that `examples` is user-facing, I'm not sure how we generally address this need outside of the `prereqs` section we call out in our examples & tutorials, e.g: https://www.pulumi.com/docs/reference/tutorials/kubernetes/tutorial-eks/#prerequisites

Reply:
Update: I'll move these from the `Makefile` to the `.travis.yml` file instead. the `aws-iam-authenticator` is actually already being installed in our [current travis config](https://github.com/pulumi/examples/blob/master/.travis.yml#L35-L36) so i'll remove these steps from the Makefile.

Reply:
Done